packages/web-components/src/components/leaving-ibm/__stories__/README.stories.react.mdx

```diff
@@ -0,0 +1,48 @@
+import { Preview, Props, Description, Story } from '@storybook/addon-docs/blocks';
+import contributing from '../../../../../../docs/contributing-license.md';
+import LeavingIbmComposite from '../../../components-react/leaving-ibm/leaving-ibm-composite';
+import filterDocgenInfoProps from '../../../../.storybook/react/filterDocgenInfoProps';
+
+# Leaving IBM
+
+> 💡 Check our
+> [CodeSandbox](https://githubbox.com/carbon-design-system/carbon-for-ibm-dotcom/tree/master/packages/web-components/examples/codesandbox/components-react/leaving-ibm)
+> example implementation.
+
+[![Edit @carbon/ibmdotcom-web-components](https://codesandbox.io/static/img/play-codesandbox.svg)](https://githubbox.com/carbon-design-system/carbon-for-ibm-dotcom/tree/master/packages/web-components/examples/codesandbox/components-react/leaving-ibm)
+
+## Getting started
+
+Here's a quick example to get you started.
+
+### JS
+
+```javascript
+import { Provider } from 'react-redux';
+import DDSLeavingIbmContainer, { store } from '@carbon/ibmdotcom-web-components/es/components-react/leaving-ibm/leaving-ibm-container.js';
+
+function App() {
+  return (
+    <Provider store={store}>
```

Comment:
Just curious, does the wrapper version need a `Provider` wrapper for it to work? Just wondering if this will confuse adopters when they try to pull this into their application and if they aren't using redux.

Reply:
@jeffchew `react-redux` uses `<Provider>` to let app control what store is bound to `react-redux` connected components: https://react-redux.js.org/api/provider. Given this is popular ["context provider" pattern in _React_](https://reactjs.org/docs/context.html#contextprovider), I our users can intuitively adapt such patterns in their application.

Reply:
Got it. We may want to add a one-liner that the `Provider` is only in the context of the example, and can be swapped out with other implementations based on the application setup.

Reply:
Makes sense - Thank you for the suggestion @jeffchew - Updated.